packages/ui-components/src/components/form/Button.vue

```diff
@@ -1,24 +1,26 @@
 <template>
-  <Component
-    :is="to ? linkComponent : 'button'"
-    :href="to"
-    :to="to"
-    :type="buttonType"
-    :external="external"
-    :class="buttonClasses"
-    :disabled="isDisabled"
-    role="button"
-    :style="
-      color !== 'subtle' && !text
-        ? `box-shadow: -1px 1px 4px 0px #0000000a inset; box-shadow: 0px 2px 2px 0px #0000000d;`
-        : ''
-    "
-    @click="onClick"
-  >
-    <Component :is="finalLeftIcon" v-if="finalLeftIcon" :class="iconClasses" />
-    <slot v-if="!hideText">Button</slot>
-    <Component :is="iconRight" v-if="iconRight || !loading" :class="iconClasses" />
-  </Component>
+  <div :class="isDisabled ? 'cursor-not-allowed' : ''">
```

Comment:
This is needed if we want to keep the `cursor-not-allowed` class, because by disabling all pointer events we also disable the option to set the cursor type

Reply:
(Oke this kills the tests)